bitrise.yml

```diff
@@ -28,7 +28,7 @@ workflows:
     - script@1:
         title: Run some verification tasks
         inputs:
-        - content: "./gradlew detektRelease lintAnalyzeDebug lintDebug lintReportDebug testDebugUnitTest jacocoDebugReport"
+        - content: "./gradlew detekt detektRelease lintAnalyzeDebug lintDebug lintReportDebug testDebugUnitTest jacocoDebugReport"
```

Comment:
Isn't `detektRelease` task redundant here?

Reply:
There was an issue (SDKCF-5622) where `detekt` and `detektRelease` gives diff results causing build failures. `gradlew check` done on release executes both of these detekt commands. As a workaround we execute both on normal builds too to prevent release inconsistencies